webpack.config.js

```diff
@@ -11,13 +11,14 @@ const files = {
 	'js/job-submission': 'js/job-submission.js',
 	'js/multiselect': 'js/multiselect.js',
 	'js/term-multiselect': 'js/term-multiselect.js',
+	'js/admin/wpjm-notice-dismiss': 'js/admin/wpjm-notice-dismiss.js',
 	'css/admin': 'css/admin.scss',
 	'css/admin-notices': 'css/admin-notices.scss',
 	'css/frontend': 'css/frontend.scss',
 	'css/job-listings': 'css/job-listings.scss',
 	'css/job-submission': 'css/job-submission.scss',
 	'css/setup': 'css/setup.scss',
-	'css/menu': 'css/menu.scss',
+	'css/menu': 'css/menu.scss'
```

Comment:
I think it's a good practice to leave the trailing comma

Reply:
This was a JSHint suggestion, but looking into it further, it seems to have been made only out of respect to old versions of IE. Reading up on 'final commas' (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas), trailing commas are indeed good practice!

Reply:
Committed https://github.com/Automattic/WP-Job-Manager/pull/2401/commits/229e6a4f410620d5e9b495cc725f8ca6e7e3a1d0